serverless.yml

```diff
@@ -0,0 +1,56 @@
+service: full-lambda-function
+
+# You can pin your service to only deploy with a specific Serverless version
+# Check out our docs for more details
+frameworkVersion: '3'
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  stage: dev
+  region: eu-west-1
+ # configValidationMode: error
+
+plugins:
+  - serverless-esbuild
+
+package:
+  include:
+    src/**/*
+  exclude:
+    - node_modules/**/*
+
+custom:
+  esbuild:
+    bundle: true
+    minify: false
+    sourcemap: true
+    exclude: 'aws-sdk'
+    target: node14
+    define:
+      'require.resolve': undefined
```

Comment:
What does this line does?

Reply:
nothing, i realize now that I could be more meticulous regarding all the code. I removed this lines.